build/android/BUILD.gn

```diff
@@ -0,0 +1,340 @@
+# Copyright (c) 2016 Intel Corporation. All rights reserved.
```

Comment:
Well, they do have to do with the source code: they all depend on other targets and files and generate something out of them afterwards. I'd put them wherever the original targets that generate libxwalkcore & co are.